### ISSUE TITLE
When do we merge all the PRs - solve the issues?

### ISSUE THREAD

**Issue by manu37** [cmsimple-xh/xhshop#228]
Wie sieht es hier mit einem neuen Release aus?
Da ich nächstens einen neuen XH-Shop aufsetzen will, wäre es schön, eine bereinigte Version zu haben, in welchem die bekannten Bugs behoben, die PR's eingeflossen sind. Wie wollen wir vorgehen?
Gruss manu

**Comment by cmb69** [comment]
Ja, version 1.1 ist lange überfällig. Vielleicht magst du den [Milestone 1.1](https://github.com/cmsimple-xh/xhshop/milestone/7) mal durchgehen und bei Bedarf korrigieren?
